app/models/article.rb

```diff
@@ -15,21 +18,32 @@ class Article < ApplicationRecord
            through: :authorships,
            dependent: :destroy,
            source: :user
-
   has_many :media, dependent: :destroy
   has_many :comments, dependent: :destroy
   has_many :outquotes, dependent: :destroy
 
-  def init
-    self.update(is_published: false)
-  end
-
   def self.order_by_rank
     Article
-      .joins("LEFT JOIN sections ON articles.section_id = sections.id")
+      .joins(:section)
       .order("articles.rank + 3 * sections.rank + 12 * articles.issue + 192 * articles.volume DESC")
   end
 
-   # TODO: generate cleaned/truncated content for summary if no summary provided
+  private
+    def init
+      if self.summary.nil? || self.summary.empty?
```

Comment:
Looks good. The only issue I see is the html tags being counted as words, but that's a little acceptable

Reply:
That shouldn't happen. [sanitize](http://api.rubyonrails.org/classes/ActionView/Helpers/SanitizeHelper.html) is built for stripping HTML tags.

Reply:
Nah but you're running the split by space thingy before you run the html tag sanitize

Reply:
Also, I'm pretty sure `"hello world".split` works like `"hello world".split(" ")`